src/docs/product/issues/performance-issues/index.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: "Performance Issues"
+sidebar_order: 15
+description: "Learn more about Performance Issues and how to use them."
+---
+
+<Include name="early-adopter-note.mdx" />
+
+The **Issue Details** page for performance issues helps you to gain further insight into the source of the issue and the impact it has on your application's users. The main area of the page displays information about a specific transaction event that's part of an issue. The top panel and right-hand sidebar of the page display a summary of all the events grouped together in this issue.
+
+<Note>
+
+You can find performance issues by searching `issue.category:performance` in **Issues**, **Discover** and **Dashboards**.
+
+</Note>
+
+![Performance issue](performance-issue-details.png)
+
+While the **Issue Details** page displays information about a specific transaction event that's part of an issue, you can navigate between the events of an issue using the "Older" and "Newer" buttons. You'll see information about the issue as well as the details of the most recent event. You'll be able to see the summary of when the issue was seen, a breakdown of the tags, and breadcrumbs if they are available.
+
+## How Performance Issues Work
+
+Sentry detects performance issues by scanning incoming transaction events. It checks various properties of the transaction (span durations, span arrangement, span types, etc.) to detect likely problems. Sentry then checks these properties against corresponding thresholds to determine whether a performance issue exists. Sentry then generates a unique problem fingerprint based on the problem type and location in the application, and uses that fingerprint to create a performance issue.
```

Comment:
to avoid repetition and calling out Sentry in the third person, switched to "we" after the first sentence

```suggestion
Sentry detects performance issues by scanning incoming transaction events. First we check various properties of the transaction (span durations, span arrangement, span types, and so on) to detect likely problems. We then check these properties against corresponding thresholds to determine whether a performance issue exists. Lastly, we generate a unique problem fingerprint based on the problem type and location in the application, and use that fingerprint to create a performance issue.
```